indexer/ftypes_matcher.cpp

```diff
@@ -693,8 +693,8 @@ class UKRoadShieldParser : public SimpleRoadShieldParser
 {
 public:
   UKRoadShieldParser(std::string const & baseRoadNumber)
-    : SimpleRoadShieldParser(baseRoadNumber, {{'M', RoadShieldType::UK_Motorway},
-                                              {'A', RoadShieldType::UK_Highway}})
+    : SimpleRoadShieldParser(
```

Comment:
I think now it's too much code for storing it in ftypes_matcher. 
I'll do a separate file for shield parsing.